pkg/proxy/register.go

```diff
@@ -116,6 +119,22 @@ func (p *Register) createDirector(proxyDefinition *Definition, balancer Balancer
 			}
 		}
 
+		parameterNames := parameterNameExtractor.Extract(path)
+		for _, parameterName := range parameterNames {
+			parameterValue := router.URLParam(req, parameterName)
+
+			if len(parameterValue) == 0 {
+				continue
```

Comment:
I'm not sure about this one, if you `id` is an empty string then the path will have `{id}` in it.

Reply:
I was in doubt as well on this, on a second look I should probably log the error to give visibility to it, but the behaviour should stay the same, the role of this component is to alter the request to proxy it somewhere else, but we cannot give an actual response from here

In the same file the approach is the same: log and return: pkg/proxy/register.go:96

What do you think?